README.md

```diff
@@ -1,2 +1,77 @@
-# URL Shortner
-URL Shortner is a Simple Web App Tool, which is used to shorten long URLs.
+<h3 align="center">URL Shortner</h3>
+
+  <p align="center">
+    URL Shortner is a Simple Web App Tool, which is used to shorten long URLs.   
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [URL Shortner](#about-the-project)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Contributing](#contributing)
+
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Tis project is about to shortner the URL with the help of python. 
+
+### Built With
+This project is build with following languages and framework
+* [Python](https:://docs.python.org/release/3.9.7/)
```

Comment:
These URLs were Perfect Earlier, `https://` is the correct one

Reply:
okay